sql-parser/src/main/antlr/SqlBase.g4

```diff
@@ -655,6 +656,8 @@ COLUMN: 'COLUMN';
 OPEN: 'OPEN';
 CLOSE: 'CLOSE';
 
+RENAME: 'RENAME';
```

Comment:
this is a breaking change if RENAME is not also added to nonReservedKeywords - either do that or add a entry in the CHANGES.txt